doc/release_notes/v2.0.0.rst

```diff
@@ -51,8 +51,8 @@ Fixes
 * Fixed error when writing elements with a VR of **OF** (:issue:`1075`)
 * Fixed improper conversion when reading elements with a VR of **OF**
   (:issue:`1075`)
-* Fixed :func:`~pydicom.pixel_data_handlers.util.apply_voi_lut` and
-  :func:`~pydicom.pixel_data_handlers.util.apply_modality_lut` not handling
+* Fixed :func:`~pydicom.pixels.apply_voi_lut` and
+  :func:`~pydicom.pixels.apply_modality_lut` not handling
```

Comment:
I don't think it makes sense to change the module names in documentation related to older versions - release notes should be a static snapshot.  If someone is using those versions, they will not see the correct module paths.  At best, can perhaps add a note in brackets noting the module name change in version 2.4.